src/fides/cli/commands/generate.py

```diff
@@ -127,7 +115,7 @@ def generate_dataset_bigquery(
 @click.pass_context
 def generate_system(ctx: click.Context) -> None:
     """
-    Generate fides System resources
+    Generate Fides systems from .
```

Comment:
Is this line missing something or is `.` supposed to be the working dir?

Reply:
If it is supposed to the working dir maybe it could be wrapped in backticks so the period is rendered as a code snippet

Reply:
typo